src/components/DappsGallery/content/DappsGalleryContent.js

```diff
@@ -0,0 +1,23 @@
+import First from '../../../../static/img/tools/universalpage_logo.png';
+import Second from '../../../../static/img/tools/stakingverse_logo.jpeg';
+import Third from '../../../../static/img/tools/universalpage_logo.png';
+import Fourth from '../../../../static/img/tools/stakingverse_logo.jpeg';
+
+export const dappsContent = [
+  {
+    project: First,
```

Comment:
the vaariable name is misleading. You need to describe what is in that variable.

Here it is a URL so a name like: `imageUrl` or `coverImageUrl` would make more sense.